lib/workflow.go

```diff
@@ -0,0 +1,138 @@
+package lib
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+)
+
+type WorkflowService service
+
+func (w *WorkflowService) List(ctx context.Context, options *WorkflowGetRequestOptions) (*WorkflowGetResponse, error) {
+	var resp WorkflowGetResponse
+	URL := w.client.config.BackendURL.JoinPath("workflows")
+	if options == nil {
+		options = &WorkflowGetRequestOptions{}
+	}
+	queryParams, _ := json.Marshal(options)
+
+	req, err := http.NewRequestWithContext(ctx, http.MethodGet, URL.String(), bytes.NewBuffer(queryParams))
+	if err != nil {
+		return nil, err
+	}
+
+	_, err = w.client.sendRequest(req, &resp)
+	if err != nil {
+		return nil, err
+	}
+	return &resp, nil
+}
+
+func (w *WorkflowService) Create(ctx context.Context, request WorkflowData) (*WorkflowData, error) {
```

Comment:
its ideal to create `WorkFlowCreatePayload` in model and use it here, otherwise the sdk method signature would confuse the users, as`WorkflowData` is a superset of the create payload